Files/Helpers/JumpListManager.cs

```diff
@@ -59,42 +60,52 @@ private void AddFolder(string path)
-                {
-                    var localSettings = ApplicationData.Current.LocalSettings;
-                    displayName = localSettings.Values.Get("RecycleBin_Title", "Recycle Bin");
-                }
-                else if (App.LibraryManager.TryGetLibrary(path, out LibraryLocationItem library))
-                {
-                    var libName = Path.GetFileNameWithoutExtension(library.Path);
-                    switch (libName)
+
+                if (displayName == null)
+				{
+                    if (path.Equals(CommonPaths.DesktopPath, StringComparison.OrdinalIgnoreCase))
+                    {
+                        displayName = "ms-resource:///Resources/SidebarDesktop";
+                    }
+                    else if (path.Equals(CommonPaths.DownloadsPath, StringComparison.OrdinalIgnoreCase))
                     {
-                        case "Documents":
-                        case "Pictures":
-                        case "Music":
-                        case "Videos":
-                            // Use localized name
-                            displayName = $"ms-resource:///Resources/Sidebar{libName}";
-                            break;
+                        displayName = "ms-resource:///Resources/SidebarDownloads";
+                    }
+                    else if (path.Equals(CommonPaths.RecycleBinPath, StringComparison.OrdinalIgnoreCase))
+                    {
+                        var localSettings = ApplicationData.Current.LocalSettings;
+                        displayName = localSettings.Values.Get("RecycleBin_Title", "Recycle Bin");
+                    }
+                    else if (App.LibraryManager.TryGetLibrary(path, out LibraryLocationItem library))
+                    {
+                        var libName = Path.GetFileNameWithoutExtension(library.Path);
+                        switch (libName)
+                        {
+                            case "Documents":
+                            case "Pictures":
+                            case "Music":
+                            case "Videos":
+                                // Use localized name
+                                displayName = $"ms-resource:///Resources/Sidebar{libName}";
+                                break;
 
-                        default:
-                            // Use original name
-                            displayName = library.Text;
-                            break;
+                            default:
+                                // Use original name
+                                displayName = library.Text;
+                                break;
+                        }
+                    }
+                    else
+                    {
+                        displayName = Path.GetFileName(path);
                     }
-                }
-                else
-                {
-                    displayName = Path.GetFileName(path);
```

Comment:
Unrelated to the PR, but here we should likely keep the newer visited folders at the top of the list or they won't be shown if there are more that ~15 items, like this:
```cs
instance.Items.Remove(instance.Items.FirstOrDefault(x => x.Arguments.Equals(path, StringComparison.OrdinalIgnoreCase)));
instance.Items.Insert(0, jumplistItem); // Keep newer items at the top
JumpListItemPaths.Remove(JumpListItemPaths.FirstOrDefault(x => x.Equals(path, StringComparison.OrdinalIgnoreCase)));
JumpListItemPaths.Add(path);
```